notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb

```diff
@@ -0,0 +1,1333 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    @component(base_image="tensorflow/tensorflow:latest")
</pre><p>I would recommend adding <code>packages_to_install=['opencv-python-headless',...]</code>  to the component decorator, rather than using <code>subprocess.call()</code> . It has the added benefit of being more readable so that users know which packages are needed on top of the base image upfront.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>